0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -56,3 +52,119 @@ All ILP Packets MUST be returned with the HTTP status code `200: OK`.
 
 An endpoint MAY return standard HTTP errors, including but not limited to: a malformed or unauthenticated request, rate limiting, or an unresponsive upstream service. Connectors SHOULD either retry the request, if applicable, or relay an ILP Reject packet back to the original sender with an appropriate [Final or Temporary error code](./0027-interledger-protocol-4/0027-interledger-protocol-4#error-codes). Server errors (status codes 500-599) SHOULD be translated into ILP Reject packets with `T00: Temporary Error` codes.
 
+### Authentication
+When authenticating requests between Interledger nodes, it is important to choose an algorithm that maintains an appropriate balance between usability and security, while at the same time maintaining high-performance _and_ interoperability.
+
+In order to find this balance, this document defines two Authentication profiles, each with various trade-offs that should be considered before use:
+
+* `SIMPLE`: Allows two ILP nodes to utilize a previously agreed-upon shared-secret as a [Bearer token](https://tools.ietf.org/html/rfc6750) in all ILP-over-HTTP requests. Peers SHOULD consider this token to be opaque and SHOULD NOT derive any special meaning from the token. 
```

Comment:
This morning it occurred to me that since this RFC has added the `Auth-Principal` header, this profile is now _exactly_ equivalent from a security perspective to HTTP BASIC Auth. Except that it uses different headers. 

This had me wondering if we shouldn't just replace the `SIMPLE` bearer scheme with HTTP Basic Auth so that implementations don't have to implement something custom -- almost every HTTP client library in every language will support BASIC auth.

Reply:
After building an implementation of this PR in Java today, it counterintuitively turns out that I can re-use the `JWT_HS_256` Authentication provider for _both_ profiles (`SIMPLE` and `JWT_HS_256`). 

This is because I can implement it once for `JWT_HS_256`, and then to support `SIMPLE`, I simply give my peer an `HS_256` JWT with a very long expiry. From my peer's perspective, this just an opaque token, which it happily re-uses for a very long time. 

From that perspective, keeping `SIMPLE` as a Bearer scheme actually reduces implementation work on the server. For the outgoing client though, it's a bit more of a pain, and using BASIC auth would be less work.

All that to say, the more I think about it, I feel like even defining `SIMPLE` is maybe a mistake. The PR is quite overt that nobody should use it in production, so I wonder if we just remove it entirely from the PR so that there's only one authentication profile that all implementations must normatively support.

Any strong thoughts or opinions about keeping or getting rid of the `SIMPLE` profile?

Reply:
Happy to get rid of it. It did occur to me on first reading it that a JWT with long expiry would probably be fine for testing and COULD be generated by the server anyway.

Reply:
> Happy to get rid of it.

I'm leaning towards removing `SIMPLE`. I will wait a day or two in order to give others a chance to comment.